maintenance/zero_downtime_migration.rst

```diff
@@ -0,0 +1,167 @@
+.. _zero_downtime_migration:
+
+Zero-downtime Migration
+===============================
+
+Firely Server v6 introduces an enhanced MongoDB database migration process. In contrast to the previous migration method that required system downtime, this new approach enables zero-downtime migration. This allows for continuous operation of your existing Firely Server instance while the data migration to the new schema proceeds.
+
+.. note::
+   This migration process applies to the *vonkdata* database. 
+   
+   If you also use MongoDB as the **admin** database, you will need to initialize a new *vonkadmin* database from scratch by specifying a different connection string for the admin database than your previous installation.
```

Comment:
I'd include a reference to :ref:`conformance` on how to initialize a fresh admin database.